components/form/index.tsx

```diff
@@ -1,26 +1,164 @@
-import s from './form.module.css'
+import {
+  // Box,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  FormHelperText,
+  Input,
+  Button,
+  Textarea,
+  Select,
+  // Checkbox,
+  Radio,
+  RadioGroup,
+} from '@chakra-ui/react'
+import { Formik, Field, Form } from 'formik'
+import * as yup from 'yup'
+import Router from 'next/router'
 
+const schema = yup.object().shape({
+  title: yup.string().required(), // TODO: length validation
+  content: yup.string().required(),
+  postal: yup.string().required(), // TODO: need to do postal FSA validation
+  category: yup.string(), // TODO: Figure out enum validation for list options
+  // anonymous: yup.bool().required(),
+  // contact: yup.bool().required(),
+  // name: yup.string(),
+  // email: yup.string().email(),
+  // phone: yup.string(), // TODO: Phone validation
+  // twitter: yup.string(),
+})
+
+type LoginFormInputs = {
+  title: string
+  content: string
+  postal: string
+  category?: string
+  // anonymous: boolean
+  // contact: boolean
+  // name?: string
+  // email?: string
+  // phone?: string
+  // twitter?: string
+}
+
+const initialValues = {
+  title: '',
+  content: '',
+  postal: '',
+  category: '',
+  anonymous: '',
+  contact: '',
+  name: '',
+  email: '',
+  phone: '',
+  twitter: '',
+}
 export default function StoryForm() {
```

Comment:
I'm keeping this as one giant component now. It'll need splitting out later certainly.